clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -99,22 +99,22 @@ export const NoticeConsentButtons = ({
 
   const handleAcceptAll = () => {
     onSave(
-      ConsentMethod.accept,
+      ConsentMethod.ACCEPT,
```

Comment:
I went through and made these all uppercased for consistency... that's a bunch of the diff here @allisonking 👍